packages/zoe/test/unitTests/contracts/test-sellTickets.js

```diff
@@ -5,7 +5,7 @@ import { test } from 'tape-promise/tape';
 // eslint-disable-next-line import/no-extraneous-dependencies
 
 import bundleSource from '@agoric/bundle-source';
-import makeIssuerKit from '@agoric/ertp';
+import { makeIssuerKit, makeLocalAmountMath } from '@agoric/ertp/src/issuer';
```

Comment:
Issuer is the default, so I believe we can just do:
```suggestion
import { makeIssuerKit, makeLocalAmountMath } from '@agoric/ertp';
```

Reply:
done pervasively